src/dftbp/reks/rekscommon.F90

```diff
@@ -430,49 +465,35 @@ end subroutine udExpand4
 
 
   !> Convert the matrix from AO basis to MO basis
-  subroutine matAO2MO(mat, eigenvecs)
+  subroutine convertMatrix(denseDesc, eigenvecs, mat, choice)
 
-    !> matrix converted from AO basis to MO basis
-    real(dp), intent(inout) :: mat(:,:)
+    !> Dense matrix descriptor
+    type(TDenseDescr), intent(in) :: denseDesc
 
     !> eigenvectors
     real(dp), intent(in) :: eigenvecs(:,:)
 
-    real(dp), allocatable :: tmpMat(:,:)
-    integer :: nOrb
-
-    nOrb = size(eigenvecs,dim=1)
-
-    allocate(tmpMat(nOrb,nOrb))
-
-    tmpMat(:,:) = 0.0_dp
-    call gemm(tmpMat, mat, eigenvecs)
-    call gemm(mat, eigenvecs, tmpMat, transA='T')
-
-  end subroutine matAO2MO
-
-
-  !> Convert the matrix from MO basis to AO basis
-  subroutine matMO2AO(mat, eigenvecs)
-
-    !> matrix converted from MO basis to AO basis
+    !> matrix to be converted
```

Comment:
```suggestion
    !> Matrix to be converted
```
See https://dftbplus-develguide.readthedocs.io/en/latest/fortranstyle.html#comments (capitalization).
This convention should be followed throughout the code changes, fortunately there is a tool so no need for manual intervention: [capitalize_ford_comments](https://github.com/dftbplus/dftbplus/blob/main/utils/srcmanip/capitalize_ford_comments).